api/src/main/java/com/nicknathanjustin/streamercontracts/votes/VoteModel.java

```diff
@@ -0,0 +1,43 @@
+package com.nicknathanjustin.streamercontracts.votes;
+
+import com.nicknathanjustin.streamercontracts.contracts.ContractModel;
+import com.nicknathanjustin.streamercontracts.users.UserModel;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import java.sql.Timestamp;
+import java.util.UUID;
+
+@Entity
+@Table(name = "votes")
+@Builder
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
+@Setter(value = AccessLevel.PACKAGE)
+@Getter
+public class VoteModel {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
+
+    @OneToOne
```

Comment:
OneToOne? There can be more than one vote for a contract right?

Reply:
Yup. My bad. good catch. fixed

Reply:
This is still showing up as OneToOne for me? Am I not loading the changes in Github properly or is it actually not changed?

#Github n00b

Reply:
This file is outdated. There should be a tag after the file name indicating this